File: itests/helloworld.java
///usr/bin/env jbang "$0" "$@" ; exit $?
// //DEPS <dependency1> <dependency2>
//NATIVE_OPTIONS -O1
//RUNTIME_OPTIONS -Dfoo=bar "-Dbar=aap noot mies"
//MANIFEST foo bar=baz baz=${bazprop:nada}

import static java.lang.System.*;

public class helloworld {

    public static void main(String... args) {
        out.println("Hello " + (args.length>0?args[0]:"World"));
    }
}

File: itests/helloworld.kt
///usr/bin/env jbang "$0" "$@" ; exit $?
// //DEPS <dependency1> <dependency2>
//NATIVE_OPTIONS -O1
//RUNTIME_OPTIONS -Dfoo=bar "-Dbar=aap noot mies"
//MANIFEST foo bar=baz baz=${bazprop:nada}

class helloworld {
    fun main(vararg args: String) {
        println("Hello " + if (args.length>0) args[0] else "World"));
    }
}
